amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -207,40 +207,48 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
             </EditableSection>
             <section className="column-layout-2">
               <section className="left-panel">
-                <div className="section-title title-3">Owners</div>
-                <div>
-                  {
-                    dashboard.owners.length > 0 &&
-                    dashboard.owners.map(owner =>
-                      <Link
-                        key={owner.user_id}
-                        to={`/user/${owner.user_id}?source=${DASHBOARD_OWNER_SOURCE}`}>
-                        <AvatarLabel
-                          label={owner.display_name}/>
-                      </Link>
-                    )
-                  }
-                  {
-                    dashboard.owners.length === 0 &&
-                    <AvatarLabel
-                      avatarClass='gray-avatar'
-                      labelClass='text-placeholder'
-                      label={NO_OWNER_TEXT}
-                    />
-                  }
-                </div>
-                <div className="section-title title-3">Created</div>
-                <div className="body-2 text-primary">
-                  { formatDateTimeShort({ epochTimestamp: dashboard.created_timestamp}) }
-                </div>
-                <div className="section-title title-3">Last Updated</div>
-                <div className="body-2 text-primary">
-                  { formatDateTimeShort({ epochTimestamp: dashboard.updated_timestamp }) }
-                </div>
-                <div className="section-title title-3">Recent View Count</div>
-                <div className="body-2 text-primary">
-                  { dashboard.recent_view_count }
-                </div>
+                <section className="metadata-section">
+                  <div className="section-title title-3">Owners</div>
```

Comment:
Maybe we could try to use an h3 or h4 here?

Reply:
I'll revisit this. If anything it would be `h4` because the existing `h3` style specifications don't match. But I want double check with design about our typography specifications. 

A couple of things I'm confused by and can't remember the context of, such as the deal with our `title-#` styles -- for example, `<h4 className="section-title title-3">` makes me question if we also need to revisit the `title-#` styles to understand what styles should be applied by default to all headers across the app.